test/cljc/rems/common/test_util.cljc

```diff
@@ -14,3 +14,7 @@
 (deftest test-distinct-by
   (is (= [1 2]
          (sort (distinct-by even? [1 2 3 4])))))
+
+(deftest test-in?
+  (is (= true (in? 1 '(1 2 3))))
+  (is (= nil (in? 4 '(1 2 3)))))
```

Comment:
Just as a sidenote,  we have preferred returning a boolean from a predicate (ends in `?`, so `false` here). This makes it slightly easier to reason about sometimes and the API to outside world is slightly more explicit. Often it makes no difference and truthy/falsey is enough.